db_access.py

```diff
@@ -1,21 +1,27 @@
 import sqlite3
-
+import datetime, hashlib
 from flask import app, g
 
 from chirpr import app
 
-def get_all_chirps():
+def hash_ps(text):
+    t = hashlib.sha224()
+    t.update(text.encode('utf-8'))
+    t = t.hexdigest()
+    return t
```

Comment:
My approach would be:

    def hash_ps(text):
        return hashlib.sha224(password.encode('utf-8')).hexdigest()

Calling `update` would be typical if you were iterating through a loop and building the string to be hashed. In this case, it doesn't seem to add much. Also, your code raises the question, "what exactly is `t`?"

The answer is, `t` is a hash instance -- until it is a hashed string. Your code here works fine, but it is not best to use the same variable to refer to very different things.